services/director-v2/src/simcore_service_director_v2/models/schemas/services.py

```diff
@@ -73,7 +73,15 @@ class RunningServiceDetails(BaseModel):
     )
     service_state: ServiceState = Field(
         ...,
-        description="the service state * 'pending' - The service is waiting for resources to start * 'pulling' - The service is being pulled from the registry * 'starting' - The service is starting * 'running' - The service is running * 'complete' - The service completed * 'failed' - The service failed to start\n",
+        description=(
+            "the service state"
+            " * 'pending' - The service is waiting for resources to start"
+            " * 'pulling' - The service is being pulled from the registry"
+            " * 'starting' - The service is starting"
+            " * 'running' - The service is running"
+            " * 'complete' - The service completed"
+            " * 'failed' - The service failed to start"
```

Comment:
where did you get this list from?

Reply:
it was always there ... please check [my answer to you](https://github.com/ITISFoundation/osparc-simcore/pull/2132#discussion_r569858326) in another PR

Reply:
maybe @sanderegg has more ideas where all these came from originally?

Reply:
they are defined in the director ... there they map to states from docker. Sorry, I still do not follow what is missing here. I guess @sanderegg  and @odeimaiz defined that map at some point while designing the front-end UI/UX